.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,68 @@
+name: 🐛 Bug Report
```

Comment:
i like this little bug emoji :D

Reply:
Yeah, I saw that on the xarray site and decided it was a nice touch